staging/src/k8s.io/kube-aggregator/pkg/controllers/openapi/aggregator/aggregator.go

```diff
@@ -197,7 +197,9 @@ func (s *specAggregator) updateServiceLocked(name string) error {
 		if err != nil {
 			return nil, "", err
 		}
-		return aggregator.FilterSpecByPathsWithoutSideEffects(result, []string{"/apis/"}), etag, nil
+		group := specInfo.apiService.Spec.Group
+		version := specInfo.apiService.Spec.Version
+		return aggregator.FilterSpecByPathsWithoutSideEffects(result, []string{"/apis/" + group + "/" + version}), etag, nil
```

Comment:
is the trailing slash the path prefix had previously (`/apis/`) significant?

the new prefix does not end in a slash... does this mean APIService group=foo, version=v1 can return `/apis/foo/v1beta1` stuff and have it included?

Reply:
yes, thanks for catching this. Opened https://github.com/kubernetes/kubernetes/pull/123770